### PR TITLE
chore: remove custom token tab

### LINKED ISSUE #2001: [Bug] Remove the "Add your own" tab in the add token dialog
### System

- [ ] Windows

  - OS Version:

- [x] Mac OS X

  - OS Version:

- [ ] Linux

  - Linux Distribution:
  - OS Version:

### Platform/Browser

- [x] Chrome

  - Maskbook Version:
  - Browser Version:

- [x] Firefox

  - Maskbook Version:
  - Browser Version:

- [ ] Android

  - Maskbook Version:
  - Android Version:

- [ ] iOS
  - Maskbook Version:
  - iOS Version:

### Build Variant

- Where do you get Maskbook?
  - [x] Store
  - [ ] ZIP
  - [ ] Self-Compiled
    - Build Commit: /_Optionally attach a Commit ID, if it is from an pre-release branch head_/

### Actual Behavior

/_What happened?_/

### PR DESCRIPTION
**Pull request by zhouhanseng**
close #2001

### CODE REVIEW

File: packages/maskbook/src/_locales/ja/messages.json [hunk 1]
```diff
@@ -11,7 +11,6 @@
     "add_to_key_store_verifying": "公開鍵が見つかりました。認証しています…",
     "add_token": "トークンを追加",
     "add_token_well_known": "身元のわかるトークン",
```
Comment (Tedko) [inline packages/maskbook/src/_locales/ja/messages.json:13]: I feel this translation in JA is wrong.

File: packages/maskbook/src/_locales/ja/messages.json [hunk 2]
```diff
@@ -11,7 +11,6 @@
...
-    "add_token_your_own": "あなたのものを追加する",
     "add_token_search_hint": "トークンを探しています。",
```
Comment (Tedko) [inline packages/maskbook/src/_locales/ja/messages.json:14]: "トークンを探しています。" change it to ->
"トークンの検索。"

File: packages/maskbook/src/_locales/ja/messages.json [hunk 3]
```diff
@@ -11,7 +11,6 @@
...
     "add_token_contract_address": "取引アドレス",
```
Comment (Tedko) [inline packages/maskbook/src/_locales/ja/messages.json:15]: remove this line. not used anymore

File: packages/maskbook/src/_locales/ja/messages.json [hunk 4]
```diff
@@ -11,7 +11,6 @@
...
     "add_token_decimals": "小数",
```
Comment (Tedko) [inline packages/maskbook/src/_locales/ja/messages.json:16]: rm this line

File: packages/maskbook/src/_locales/ja/messages.json (hunk 1 above)
Comment (Tedko) [reply packages/maskbook/src/_locales/ja/messages.json:13]: we've updated the Eng i18n;
Change this to `  "add_token_well_known": "検索" `
cc @zhouhanseng

File: packages/maskbook/src/_locales/zh/messages.json [hunk 5]
```diff
@@ -11,7 +11,6 @@
     "add_to_key_store_verifying": "Maskbook 公鑰已獲取，正在驗證…",
     "add_token": "添加代幣",
     "add_token_well_known": "知名代幣",
```
Comment (Tedko) [inline packages/maskbook/src/_locales/zh/messages.json:13]: change it to 
`    "add_token_well_known": "搜索",`

File: packages/maskbook/src/_locales/zh/messages.json [hunk 6]
```diff
@@ -11,7 +11,6 @@
...
-    "add_token_your_own": "自定義代幣",
     "add_token_search_hint": "搜索 ERC20 代幣",
     "add_token_contract_address": "合約地址",
     "add_token_decimals": "小數點",
```
Comment (Tedko) [inline packages/maskbook/src/_locales/zh/messages.json:16]: will this i18n be used anymore? if not- rm it

File: packages/maskbook/src/_locales/en/messages.json [hunk 7]
```diff
@@ -11,7 +11,6 @@
     "add_to_key_store_verifying": "The public key has been found, verifying…",
     "add_token": "Add Token",
     "add_token_well_known": "Search",
-    "add_token_your_own": "Add your own",
     "add_token_search_hint": "Search Tokens",
     "add_token_contract_address": "Contract Address",
```
Comment (Tedko) [inline packages/maskbook/src/_locales/en/messages.json:15]: rm it

**Review by Tedko — CHANGES_REQUESTED** [review]

**Comment by Tedko** [comment]
check out and fix the i18n linter @zhouhanseng

File: packages/maskbook/src/_locales/ja/messages.json (hunk 1 above)
Comment (zhouhanseng) [reply packages/maskbook/src/_locales/ja/messages.json]: unused, removed

**Comment by zhouhanseng** [comment]
done

Suji Yan <notifications@github.com> 于2020年12月14日周一 下午12:36写道：

**Review by guanbinrui — APPROVED** [review]

**Comment by Tedko** [comment]
@zhouhanseng

"トークンを探しています。" change it to ->
"トークンの検索。"

How abt this

**Comment by zhouhanseng** [comment]
oh.. done: https://github.com/DimensionDev/Maskbook/pull/2100

**Pull request merged**